Maths/test/Determinant.test.js

```diff
@@ -50,7 +51,11 @@ describe('Determinant', () => {
   test.each(testCases)(
     'Should return the determinant of the square matrix.',
     (matrix, expected) => {
-      expect(determinant(matrix)).toEqual(expected)
+      try {
+        expect(determinant(matrix)).toEqual(expected)
+      } catch (err) {
+        expect(err.message).toEqual(expected)
```

Comment:
I think it would be cleaner to give the error cases their own `test.each`.

Reply:
Sure, modifying as requested.